usaspending_api/awards/models.py

```diff
@@ -191,6 +191,7 @@ class Award(DataSourceTrackedModel):
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
     latest_submission = models.ForeignKey(SubmissionAttributes, null=True)
+    latest_transaction = models.ForeignKey("awards.Transaction", related_name="latest_transaction", null=True)
```

Comment:
We have to have a related name here because there would be a collision with transaction set related names (you'll see this if you try to makemigrations without a related name) I went for 'latest_transaction' here, because I thought it was clear, however I see what you mean now. We could probably do with naming it "latest_for_award", how does that sound?